src/workflows/airqo_etl_utils/datautils.py

```
@@ -67,6 +67,9 @@ def get_devices(
         keys: Dict = {}
         devices = DataUtils._load_devices_from_cache(local_file_path)
 
+        if Config.ENVIRONMENT == "production":
+            devices = devices.loc[devices.deployed == True]
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `devices.deployed:` for truth checks

Replace with `devices.deployed`

(E712)